sjb/config/common/test_cases/ovn-kubernetes.yml

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
How is this different from e.g. the CRI-O common base?

Reply:
Just as much as artifacts are different. And we don't care about some journal files to be captured. Later on, I expect some local files to be gathered.
It won't harm to use crio at all, just would be surplus processing. Should we scrap this file to save clutter in the repo? Pretty sure we might need some customizations later on though.

Reply:
If it's possible, you could inherit from this one and just provide `overrides:` for the artifacts and debug journal files.